alws/crud/sign_key.py

```diff
@@ -50,21 +65,34 @@ async def create_sign_key(
         raise SignKeyAlreadyExistsError(
             f"Key with keyid {payload.keyid} already exists"
```

Comment:
Thanks for you comments, i'll drop the constraints 
Regarding your first comment about `SignKeyAlreadyExistsError`
Do I understand it right that right now we can only add a new sign key to a platform only in `create_sign_key()` function? Maybe it would be a better approach to create a separate function that would allow to do that? without mixing a 'create' method with basically an 'update' operation

Reply:
My bad, I totally forgot that these changes bring many-to-many relationship for sign keys, you can forget about comments above because we can just associate existing keys to platforms.
Regarding association existent keys to platforms, we don't have an interface for that now, right?
Could you create an issue for that?